plugins/modules/ipa_pwpolicy.py

```diff
@@ -64,6 +64,21 @@
     lockouttime:
         description: Period (in seconds) for which users are locked out.
         type: str
+    gracelimit:
+        description: Maximum number of ldap logins after password expiration.
+        type: str
+    maxrepeat:
+        description: Maximum number of same consecutive characters.
+        type: str
+    maxsequence:
+        description: The max. length of monotonic character sequences (abcd).
+        type: str
```

Comment:
This and the above two seem to be integers, not strings. What is the reason to use strings in the module interface?

Wouldn't it be better to make these ints, and if the API needs strings, convert them to strings in the module's code?

Reply:
I strongly agree with you, but for now I think to done this without more changes on the module.

Because as far as I see this module will require more refactors than convert str to ints.

Reply:
I will try to do this in another PR.

Reply:
All strings changed to integers @felixfontein PTAL.